test/controllers/api/v2/ostree_branches_controller_test.rb

```diff
@@ -55,5 +55,14 @@ def test_compare
       assert_response :success
       assert_template "katello/api/v2/ostree_branches/compare"
     end
+
+    def test_branch_order
+      @branch2 = @repo.ostree_branches.create!(:name => "def456", :uuid => "456uvw", :version => "1.0")
+      @default_sort = %w(version desc)
+
+      get :index, params: { :version => [@branch.version, @branch2.version] }
+      results = JSON.parse(response.body)['results'][0]['uuid']
```

Comment:
In a controller test I would expect a variable called `results` to have the response of the controller API call in its entirety. Here you've taken that response and filtered it further to take the first item out and grab the uuid field from it - so I would say the variable is misnamed.

I think a better name would be something like 'expected' or 'expected_uuid' to be super clear on what data is held inside. I know it might sound silly but when you're trying to debug a 50 line method with a lot of local variables that don't have *very good* names it can make the task unnecessarily challenging. Better to carry best practices everywhere - even in small methods like this!